api/src/main/java/org/hawkular/apm/api/model/trace/Trace.java

```diff
@@ -269,7 +271,7 @@ public long calculateDuration() {
             long elapsedTime = endTimeNS - getNodes().get(0).getBaseTime();
 
             // Convert elapsed time to milliseconds
```

Comment:
This comment, and the return value comments in calculateDuration and endTime methods should be changed as saying milliseconds.